addons/pos_restaurant/static/src/overrides/components/product_screen/order_summary/order_summary.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates id="template" xml:space="preserve">
+    <t t-name="pos_restaurant.OrderSummary" t-inherit="point_of_sale.OrderSummary" t-inherit-mode="extension">
+        <xpath expr="//Orderline" position="attributes">
+            <attribute name="t-on-dblclick">
+                () => line.toggleSkipChange()
+            </attribute>
```

Comment:
```suggestion
            <attribute name="t-on-dblclick" value="() => line.toggleSkipChange()" />
```

Check if this works. We don't want the content of the `attribute` to be translated.